### PR TITLE
LPS-133419 | master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-133419

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/sitenavigation/navigationmenu/NavigationMenus.testcase [hunk 1]
```diff
@@ -180,6 +180,100 @@ definition {
...
+		}
+
+		task ("Add the parents and children pages to the navigation menu") {
+			NavigationMenusAdmin.addItem(
+				item = "Page",
+				pageNames = "Test Page Name 1,Child Page 1,Test Page Name 2,Child Page 2,Test Page Name 3,Child Page 3");
+
+			for (var pageName : list "Test Page Name 1,Child Page 1,Test Page Name 2,Child Page 2,Test Page Name 3,Child Page 3") {
+				NavigationMenusAdmin.viewItem(itemName = "${pageName}");
+			}
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/sitenavigation/navigationmenu/NavigationMenus.testcase:227]: Use json macro to add site navigation menu items

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/sitenavigation/navigationmenu/NavigationMenus.testcase [hunk 2]
```diff
@@ -180,6 +180,100 @@ definition {
...
+		}
+
+		task ("Change the order of the third page") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/sitenavigation/navigationmenu/NavigationMenus.testcase:230]: Change the order of the second parent page item and third parent page item

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/sitenavigation/navigationmenu/NavigationMenus.testcase [hunk 3]
```diff
@@ -180,6 +180,100 @@ definition {
...
+			DragAndDrop.javaScriptDragAndDropToObject(
+				key_itemName = "Test Page Name 2",
+				key_sourceItemName = "Test Page Name 3",
+				locator1 = "SiteNavigationMenu#ITEM_ENTRY_DRAG_ICON",
+				locator2 = "SiteNavigationMenu#ITEM_ENTRY");
+		}
+
+		task ("View the pages and children still are on the right indentation") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/sitenavigation/navigationmenu/NavigationMenus.testcase:238]: View the hierachy of page items

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/sitenavigation/navigationmenu/NavigationMenus.testcase [hunk 4]
```diff
@@ -180,6 +180,100 @@ definition {
...
+				NavigationMenusAdmin.viewItemNested(
+					groupName = "Test Site Name",
+					itemName = "Child Page ${n}",
+					parentItemName = "Test Page Name ${n}",
+					site = "true",
+					siteNavigationMenuName = "Navigation Menu Name");
+			}
+		}
+
+		task ("Put the third page on its original place") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/sitenavigation/navigationmenu/NavigationMenus.testcase:253]: Change back to the original order of the second parent page item and third parent page item

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/sitenavigation/navigationmenu/NavigationMenus.testcase [hunk 5]
```diff
@@ -180,6 +180,100 @@ definition {
...
+			DragAndDrop.javaScriptDragAndDropToObject(
+				key_itemName = "Test Page Name 3",
+				key_sourceItemName = "Test Page Name 2",
+				locator1 = "SiteNavigationMenu#ITEM_ENTRY_DRAG_ICON",
+				locator2 = "SiteNavigationMenu#ITEM_ENTRY");
+		}
+
+		task ("View the pages and children still are on the right indentation") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/sitenavigation/navigationmenu/NavigationMenus.testcase:261]: View the hierachy of page items

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/sitenavigation/navigationmenu/NavigationMenus.testcase [hunk 6]
```diff
@@ -180,6 +180,100 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-125802. The user can change the order of the page and child without affect the indentation in Navigation Menu."
+	@priority = "4"
+	test ChangeOrderOfPagesInNagivationMenu {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/sitenavigation/navigationmenu/NavigationMenus.testcase:185]: ViewHierarchyOfItemsWhenChangeOrder

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/sitenavigation/navigationmenu/NavigationMenus.testcase [hunk 7]
```diff
@@ -180,6 +180,100 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-125802. The user can change the order of the page and child without affect the indentation in Navigation Menu."
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/sitenavigation/navigationmenu/NavigationMenus.testcase:183]: View the hierarchy of items when change the order

**Comment by Tim-Cao** [comment]
Hi @ozysouza, Could you update the case according to comment? Thanks

**Comment by ozysouza** [comment]
> Hi @ozysouza, Could you update the case according to comment? Thanks

Hi @Tim-Cao  PR Updated following your comments.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/sitenavigation/navigationmenu/NavigationMenus.testcase [hunk 8]
```diff
@@ -552,6 +552,119 @@ definition {
...
+			for (var layoutName : list "Test Page Name 1,Test Page Name 2,Test Page Name 3") {
+				JSONSitenavigation.addSiteNavigationMenuItem(
+					groupName = "Test Site Name",
+					parentSiteNavigationMenuItem = "${layoutName}",
+					privateLayout = "false",
+					siteNavigationMenuItemName = "Child Page ${n}",
+					siteNavigationMenuName = "Navigation Menu Name",
+					type = "layout");
+
+				var n = ${n} + 1;
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/sitenavigation/navigationmenu/NavigationMenus.testcase:609]: Simplify the parameter

**Comment by Tim-Cao** [comment]
Hi @ozysouza  see changes https://github.com/liferay-echo/liferay-portal/compare/8452bd1779cbebab35e9c2c971088f7bb12c4bc4...1d2e843a5b75d789370485685b6080689ccdc409
in https://github.com/liferay-echo/liferay-portal/pull/5121

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**